rust/Makefile.am

```diff
@@ -27,36 +28,43 @@ if RUST_CROSS_COMPILE
 RUST_TARGET = --target $(host_triplet)
 endif
 
-all-local:
-if HAVE_PYTHON
-	cd $(top_srcdir)/rust && $(HAVE_PYTHON) ./gen-c-headers.py
+all-local: $(abs_top_builddir)/rust/Cargo.lock
+if !HAVE_RUST_HEADERS
+	cbindgen --config $(abs_top_srcdir)/rust/cbindgen.toml \
```

Comment:
I think it would be good to `echo` something if `HAVE_RUST_HEADERS` is true. E.g. "using existing Rust bindings in `rust/gen`", maybe even with a small hint of how to update/remove.

Reply:
> I think it would be good to `echo` something if `HAVE_RUST_HEADERS` is true. E.g. "using existing Rust bindings in `rust/gen`", maybe even with a small hint of how to update/remove.

Thoughts on what the hint would be? There are no real specific commands (yet) to update/remove the generated headers.